lib/vanagon/platform/solaris_10.rb

```diff
@@ -48,7 +48,6 @@ def generate_package(project) # rubocop:disable Metrics/AbcSize
 
           bom_install,
 
-          "rm #{name_and_version}.tar.gz",
```

Comment:
You may run into issues here. IIRC when building some of the bigger/more complex packages (gcc), either the pooler or hardware ran out of space. Shouldn't be hard to address on the hardware/vm side, but just a heads up.

Reply:
ok, cool, thanks @haus

Reply:
I tested a gcc build on sol10 which seemed fine (failed with perm issues shipping the artifacts, but no failures before that). Do you know if there was anything else that might've triggered this?

Reply:
might also depend on the tmpdir used (which i believe has changed over time). might not be worth worrying over yet